README.md

```diff
@@ -71,13 +71,19 @@ The framework relies on [Node.js](https://nodejs.org/en/download/) and [npm](htt
     gulp generate_sample_users
     gulp generate_sample_data
 
+    # Generate Cognito User Groups
```

Comment:
i made the generate_sample_groups, and generate_sample_users idempotent. If the user or the group exists we are not trying to recreate them. FWIW, the assign users to group was idempotent by itself